.github/configs/sdkconfig.defaults

```diff
@@ -18,17 +18,7 @@ CONFIG_ETH_SPI_ETHERNET_W5500=y
 CONFIG_ETH_SPI_ETHERNET_KSZ8851SNL=y
 
 # Enable BT & Bluedroid support
-CONFIG_BT_ENABLED=y
-CONFIG_BT_BLUEDROID_ENABLED=y
-CONFIG_BT_CLASSIC_ENABLED=y
-CONFIG_BTDM_CTRL_MODE_BLE_ONLY=n
-CONFIG_BTDM_CTRL_MODE_BR_EDR_ONLY=y
-CONFIG_BTDM_CTRL_MODE_BTDM=n
-CONFIG_BT_A2DP_ENABLE=y
-CONFIG_BT_HFP_ENABLE=y
-CONFIG_BT_HFP_CLIENT_ENABLE=y
-CONFIG_BT_HFP_AUDIO_DATA_PATH_HCI=y
-CONFIG_BT_BLE_42_FEATURES_SUPPORTED=y
+CONFIG_BT_ENABLED=n
```

Comment:
Please restore the BT settings, but commented out, and put a short comment on top: "Uncomment to enable Classic BT"